packages/react-lowcode/src/codegen/interfaces.ts

```diff
@@ -14,4 +15,24 @@ export interface InsertOptions {
 
 export interface DeleteOptions {
     index: number
+}
+
+export interface ColumnSourcePositionOptions{
+    index: number
+}
+
+export interface ColumnSourcePositionResult {
+    columnPosition: SourceLineCol,
+    valuePosition?: SourceLineCol,
+    headerPosition?: SourceLineCol
+}
+
+export interface WidgetProperty {
+    name: string,
+    value?: string,
+    isWithoutValue: boolean
```

Comment:
please explain this? does it mean undefined, not set or something else?

Reply:
this will be renamed to some more meaningful name in future, it is because of `required` which basically has no value